tests/components/mastodon/snapshots/test_sensor.ambr

```diff
@@ -0,0 +1,301 @@
+# serializer version: 1
```

Comment:
Please delete the file and regenerate them, you have some leftovers from when you didn't generate translations yet